setup.py

```diff
@@ -24,6 +25,6 @@
     ],
     url='https://github.com/transifex/transifex-python',
     install_requires=[
-        'pyseeyou', 'pytz', 'requests', 'six', 'click'
+        'pyseeyou', 'pytz', 'requests', 'six', 'click', 'parsimonious'
```

Comment:
Is `parsimonious` necessary for some reason?

Reply:
Not you are right!